### ISSUE TITLE
FAWE buggy in 1.7.10

### ISSUE THREAD

**Issue by MrPowerGamerBR** [boy0001/FastAsyncWorldedit#348]
# Bug report template (Follow this template unless you are making a feature request.)
**Debug paste link**:
http://hastebin.com/ezegivadaj

**Description of the problem:**
When I try to use "//set stone", this happens

![http://i.imgur.com/2qRXuWm.png](http://i.imgur.com/2qRXuWm.png)

Relogging fixes the issue

**How to replicate**:
Use any FAWE command (//set stone, for example)
Make sure you've completed the following steps (put an X between of brackets):
- [X] Include `/fawe debugpaste`
- [X] Made sure there aren't duplicates of this report [(Use Search)](https://github.com/boy0001/FastAsyncWorldedit/issues?utf8=%E2%9C%93&q=is%3Aissue)
- [X] Made sure you're using an updated version of FAWE
- [X] Made sure the bug/error isn't caused by any other plugin

**Comment by cyberpwnn** [comment]
Try ```//fixlighting```. I have managed force that to happen when a player gets stuck inside of an invisible chunk. However my issue is it keeps coming back when you fly in and back out of your own view distance. This seems to be the chunk packets since i have tested this with another player and they dont always see them after they were fixed clientside. Here is what mine look like.

![image](https://cloud.githubusercontent.com/assets/10619135/20777700/ac589864-b736-11e6-9eb5-a0bcc364f69d.png)

This was a pasted schematic and all four corners of the pasted area have this issue every time i fly into view distance (chunks are already loaded)
